scan/lib/scan/options.rb

```diff
@@ -269,6 +269,12 @@ def self.available_options
                                      optional: true,
                                      is_string: false,
                                      default_value: false),
+        FastlaneCore::ConfigItem.new(key: :additional_xctestrun,
+                                     env_name: "SCAN_SHOULD_ADDITIONAL_XCTESTRUN",
+                                     description: "Should provide additional copy of .xctestrun file (settings.xctestrun) and place in output path?",
+                                     optional: true,
```

Comment:
Add `type: Boolean`

Reply:
Done.
Lane generate_swift_api is ok.
I have seen that some other arguments do not have a "type", so I thought that parameter "is_string" is enough.
Could you please retry CI jobs, locally is everything ok.

Reply:
Try to add a test case that addresses the changes if you can 🙏